packages/discovery-react-components/src/components/CIDocument/__stories__/default.md

```diff
@@ -0,0 +1,3 @@
+#### Overview
+
+The CIDocument component provides Element Classification to the user. It takes a PDF and enrich it with features a customer might deem important (such as obligation, Deliverables, Assignments, etc...). This would help a customer get a better understanding of their document or help them find what they need in a large document in a more efficient way.
```

Comment:
Backticks around `CIDocument` would make it more consistent with other components.

Reply:
> It takes a PDF and ~enrich~ enriches

Reply:
Done ✅